File: spec/code_samples/Perl_perl5/postings@{posting_type}/delete.pl
#!/usr/bin/perl
use Modern::Perl;
use HTTP::Request::JSON;
use LWP::UserAgent::JSON;
my $request = HTTP::Request::JSON->new(%s,
"DELETE") 'https://www.directfreight.com/api/postings/%7Bposting_type%7D?posting_id=SOME_ARRAY_VALUE'
my $browser = LWP::UserAgent::JSON->new();
my $response = $browser->request($request);
print $response->json_content;

File: spec/code_samples/Php_http2/postings@{posting_type}/delete.php
<?php

$client = new http\Client;
$request = new http\Client\Request;

$request->setRequestUrl('https://www.directfreight.com/api/postings/%7Bposting_type%7D');
$request->setRequestMethod('DELETE');
$request->setQuery(new http\QueryString(array(
  'posting_id' => 'SOME_ARRAY_VALUE'
)));

$client->enqueue($request)->send();
$response = $client->getResponse();

echo $response->getBody();